OptimizelySDK/Utils/Validator.cs

```diff
@@ -23,6 +23,14 @@ namespace OptimizelySDK.Utils
 {
     public static class Validator
     {
+        public static readonly double OPT_NUMBER_LIMIT;
+
+        static Validator()
+        {
+            OPT_NUMBER_LIMIT = Math.Pow(2, 53);
```

Comment:
Should this not be a `const`?

Reply:
No, it can't. Computing values can't be constant.

Reply:
Why does it have to be computed? 2^53 is `9007199254740992` I think?

`public static const double OPT_NUMBER_LIMIT = 9007199254740992`

Reply:
Oh, in c# constants can be the result of expressions, I think. So, this would be valid:
`public const int c2 = c1 + 100;`

shouldnt you be able to do `Math.Pow()`? I'm actually curious

Reply:
Nope that's not the case, just because of this long value of 2^53 and sync with other SDKs, I added here in this way but changed it now and made comment.